staging/src/k8s.io/apiserver/pkg/endpoints/filterlatency/filterlatency.go

```diff
@@ -64,7 +65,7 @@ func TrackCompleted(handler http.Handler) http.Handler {
 		latency := completedAt.Sub(fr.startedTimestamp)
 		metrics.RecordFilterLatency(ctx, fr.name, latency)
 		if klog.V(3).Enabled() && latency > minFilterLatencyToLog {
-			httplog.AddInfof(ctx, "%s=%s", fr.name, latency.String())
+			httplog.AddKeyValue(ctx, fmt.Sprintf("fl_%s", fr.name), latency.String())
```

Comment:
@MikeSpreitzer I also took the liberty to:
- use `AddKeyValue` 
- and add a prefix of `fl` to denote filter latency.  so we see something like `fl_priorityandfairness="122.622271ms"`

Let me know if it is okay, otherwise we can make this change in a separate PR

Reply:
Using AddKeyValue here looks good to me.
Using `fmt.Sprintf` to concatenate two strings looks unnecessarily heavyweight to me.